homeassistant/components/philips_js/media_player.py

```diff
@@ -146,38 +149,38 @@ def state(self):
     @property
     def source(self):
         """Return the current input source."""
-        if self.media_content_type == MEDIA_TYPE_CHANNEL:
-            name = self._channels.get(self._tv.channel_id)
-            prefix = PREFIX_CHANNEL
-        else:
-            name = self._sources.get(self._tv.source_id)
-            prefix = PREFIX_SOURCE
-
-        if name is None:
-            return None
-        return prefix + PREFIX_SEPARATOR + name
+        return self._sources.get(self._tv.source_id)
 
     @property
     def source_list(self):
         """List of available input sources."""
-        complete = []
-        for source in self._sources.values():
-            complete.append(PREFIX_SOURCE + PREFIX_SEPARATOR + source)
-        for channel in self._channels.values():
-            complete.append(PREFIX_CHANNEL + PREFIX_SEPARATOR + channel)
-        return complete
+        return list(self._sources.values())
 
     def select_source(self, source):
         """Set the input source."""
         data = source.split(PREFIX_SEPARATOR, 1)
         if data[0] == PREFIX_SOURCE:
+            _LOGGER.warning(
```

Comment:
Thinking about these warnings. If we never remove this code, we would also not need these warnings. Like, it doesn't hurt us to keep this code around and we can avoid a breaking change.

Reply:
i was hoping we could remove it at some stage which is why i added the warning